conda/dev.yml

```diff
@@ -6,7 +6,8 @@ dependencies:
   # Base
   # =================
   - pip=22.2.2
-  - python=3.9.13
+  - python=3.11
+  - sqlite
```

Comment:
@tomvothecoder How do we determine the constraints (`=`, `>=`, etc.)?

Reply:
This is based on the conda-forge recipe and compatibility with other packages. If things break with new package versions, that suggests updating the constraints accordingly.